packaging/torchvision/conda_build_config.yaml

```diff
@@ -1,3 +1,6 @@
+channel_sources:
+  - defaults,conda-forge
```

Comment:
is it now safe to use `conda-forge` as well? At some point we had issues with it.

Reply:
It seems only libturbojpeg is being pulled from there, so all other packages are being pulled from defaults or the main PyTorch channel